packages/sado-connect/vite.config.ts

```diff
@@ -37,5 +38,11 @@ export default defineConfig({
     }),
     commonjs(),
     cssInjectedByJsPlugin(),
+    nodePolyfills({
+      // Whether to polyfill specific globals.
+      globals: {
+        Buffer: true, // can also be 'build', 'dev', or false
```

Comment:
Don't use a `Buffer` polyfill. The browser polyfill is not identical to the Node.js `Buffer`.

https://github.com/sadoprotocol/sado-connect/commit/75681f7a5a6969325ee4d992144d5e5e2ac6af3f

Reply:
hmm, how can I test the difference? like which part will break it?
I am using this because the `buffer` doesn't work on browser like if you `pnpm dev` it breaks without this

Reply:
Your vite plugin alternative seems to work. I was able to send BTC from a legacy and nested segwit address successfully. (I hope my prior Buffer hack wasn't cached).

Let's go with your solution.